### PR TITLE
Refactor Accordion component & updates to search and select field

### PR DESCRIPTION
**Pull request by transitorytammy**

### CODE REVIEW

File: addon/components/bourbon-search-select-field.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default Component.extend(SelectMixin, {
     if (this.get("label") !== null && this.get("value") !== null) {
       this.inputValueObserver();
     } else if (this.get("value")) {
-      this.setLabel(this.get('value'));
+      this.setLabel(this.get("value"));
```
Comment (mshappe) [inline addon/components/bourbon-search-select-field.js]: Why the change? It shouldn't matter in this context.

Comment (transitorytammy) [reply addon/components/bourbon-search-select-field.js]: @mshappe are you talking about the single quote to double quote?  My editor usually fixes this when that line is touched.

Comment (lawrencegetfeedback) [reply addon/components/bourbon-search-select-field.js]: Probably just for consistency, though I do still see some single quotes being used. Might want to establish a code style guide, even if just a de facto one.

File: addon/components/bourbon-search-select-field.js [hunk 2]
```diff
@@ -134,30 +137,55 @@ export default Component.extend(SelectMixin, {
     document.activeElement.blur();
   },
 
+  getSearchString(selectedValue) {
```
Comment (mshappe) [inline addon/components/bourbon-search-select-field.js]: Clever!

File: addon/components/bourbon-search-select-field.js (hunk 1 above)
Comment (transitorytammy) [reply addon/components/bourbon-search-select-field.js]: I like the idea of following the Airbnb style guide which suggests single quotes.  Will update the repo to use single quotes and update the PR accordingly.  

https://github.com/airbnb/javascript#strings--quotes

**Comment by transitorytammy** [comment]
.5 days in January
